sw/device/tests/BUILD

```diff
@@ -355,12 +329,7 @@ opentitan_functest(
 opentitan_functest(
     name = "flash_ctrl_idle_low_power_test",
     srcs = ["flash_ctrl_idle_low_power_test.c"],
-    verilator = verilator_params(
-        tags = [
-            "cpu:4",
-            "failing_verilator",
-        ],
-    ),
+    verilator = verilator_params(tags = ["broken"]),
```

Comment:
This test (and perhaps others doesn't support FPGA or verilator IIRC,) @timothytrippel is that correct? can we mark it as dv only instead?

Reply:
It should work on FPGA an Verilator as there is no host side SystemVerilog component, see [here](https://cs.opensource.google/opentitan/opentitan/+/master:hw/top_earlgrey/dv/chip_sim_cfg.hjson;drc=33bde395b718432bb874108e0df6184a4725a5f4;l=380), the `uvm_test_seq` is set to `chip_sw_base_vseq` meaning the default sequence that has no special behavior.